components/List.tsx

```diff
@@ -1,19 +1,115 @@
-import * as React from 'react'
+import React, { useEffect } from 'react'
 import ListItem from './ListItem'
 import { User } from '../interfaces'
+import axios from 'axios';
 
 type Props = {
   items: User[]
 }
+const List = ({ items }: Props) => {
+  useEffect(() => {
+    const playlistId = "2PF13ZUdpgZgdCNtOZ3zCv";
+    getPlaylist(playlistId);
+  });
 
-const List = ({ items }: Props) => (
-  <ul>
-    {items.map((item) => (
-      <li key={item.id}>
-        <ListItem data={item} />
-      </li>
-    ))}
-  </ul>
-)
+  const setExistingSongs = (playlistSongArr) => {
```

Comment:
Are you able to output your results to the UI side? Like for example list out all the songs in a playlist or list all the users playlists - are you able to use the access token generated from next auth to run these backend functions?

Reply:
Also just to confirm this local storage code had to be moved to the UI side instead of in the API side, right?